### ISSUE TITLE
Peer review for proposal - Group G

### ISSUE THREAD

**Issue by khaukhau** [mtuann/comp5120-data-vis#2]
1. **Clarity & Plan:** The summary of a dataset of the proposal is quite clear and easy to understand.
2. **Plot options:** Although line plots, bar charts, and stacked area plots are mentioned, the proposal could benefit from a wider range of visualization techniques to better uncover and illustrate complex patterns and relationships within the data.
3. **Questions:**
* Each question should have two figures (according to the plan), so it is better to refine questions, so that it is big enough to be answered by 2 figures
* For instance, question 1: "How does the composition of collected trash vary across different Trash Wheels over time?". From a big topic, the team can come up with smaller questions, for example: "How does the volume and weight of collected waste vary seasonally?". This approach may help the group understand the dataset comprehensively and have a structured answer.
* Some variables are underexploited (ex: HomesPowered), Thus, the team can look into this variable in comparison with other variables for more insights.
4. **Feasibility:** The questions are all feasible, but the scope and approach can be expanded into more variables (explained in the plot options and questions section)

**Comment by mtuann** [comment]
Thank you for your review. We can consider to give more detail for question 1 and question 2 as below:

**Question 1:** How does the composition of collected trash vary across different Trash Wheels over time?

Refinement: How does the volume and weight of collected waste vary seasonally?

- Create a line plot to visualize the seasonal trend of volume and weight of collected waste across different months or years.
- This figure will provide insights into the seasonal patterns of waste accumulation, allowing for comparisons between Trash Wheels.

What is the correlation between the volume/weight of collected waste and the number of homes powered?

- Generate a scatter plot to explore the relationship between the volume/weight of collected waste and the number of homes powered.
- This figure will help understand the potential energy generation from waste and its relation to waste volume/weight, providing insights into the environmental impact of waste collection.

**Question 2:** What is the seasonal pattern of trash accumulation, and are there correlations between different types of trash?

Refinement: How does the seasonal pattern of trash accumulation vary for different types of trash (e.g., plastic bottles, glass bottles, plastic bags)?

- Create a line plot or bar chart to visualize the seasonal trends of different types of trash accumulation across different months or years.
This figure will show the seasonal patterns of specific types of trash accumulation, allowing for comparisons and identifying any distinct patterns.

- Is there a correlation between the accumulation of plastic bags and other types of trash (e.g., plastic bottles, cigarette butts)?

- Generate a heatmap or correlation matrix to explore the relationships between the accumulation of plastic bags and other types of trash.

This figure will help identify any significant correlations between different types of trash accumulation, providing insights into potential sources or behaviors leading to littering.

By refining the questions and focusing on specific aspects of the dataset, each question can be adequately addressed by two figures.

**Comment by tienvu95** [comment]
I agree that all proposals should consider more types of chart. If we are going to use just bar and line charts, I don't think we need to introduce additional chart types thru lectures and AE. PLus they should keep in mind the requirements of using 2 charts for each qn + explore more variables (it's always more interesting to examining data from more than 2 dimensions :D)
